src/test/java/it/pagopa/pn/bff/pnclient/userattributes/PnUserAttributesClientImplTest.java

```diff
@@ -38,6 +38,40 @@ class PnUserAttributesClientImplTest {
     @MockBean(name = "it.pagopa.pn.bff.generated.openapi.msclient.user_attributes.api.LegalApi")
     private LegalApi legalApi;
 
+
```

Comment:
Write tests also in PnUserAttributesClientImplTestIT

Reply:
write it